packages/nimble/R/cppDefs_core.R

```diff
@@ -230,6 +230,10 @@ cppCodeBlock <- setRefClass('cppCodeBlock',
                                           cppADCode = 'ANY', generatorSymTab = 'ANY'),			#'logical'),
                             methods = list(
                                 generate = function(indent = '', ...) {
+                                    if(identical(cppADCode, TRUE)){
```

Comment:
Why does it not suffice to simplify to `if (cppADCode)` ?

Reply:
If `cppADCode` is some malformed object, then `if(cppADCode)` will error instead of being handled as `FALSE`.

R provides `isTRUE(cppADcode)` as an option.